source/operations/install-deploy-manage/decommission-server-pool.rst

```diff
@@ -1,31 +1,28 @@
 .. _minio-decommissioning:
 
-==========================
-Decommission a Server Pool
-==========================
+=========================
+Decommission Server Pools
+=========================
 
 .. default-domain:: minio
 
 .. contents:: Table of Contents
    :local:
    :depth: 1
 
-Starting with :minio-release:`RELEASE.2022-01-25T19-56-04Z`, MinIO supports
-decommissioning and removing a :ref:`server pool <minio-intro-server-pool>`
-from a deployment. Decommissioning is designed for removing an older server pool
-whose hardware is no longer sufficient or performant compared to the pools in
-the deployment. MinIO automatically migrates data from the decommissioned pool
-to the remaining pools in the deployment based on the ratio of free space
-available in each pool.
-
-During the decommissioning process, MinIO routes read operations (e.g. ``GET``,
-``LIST``, ``HEAD``) normally. MinIO routes write operations (e.g. ``PUT``,
-versioned ``DELETE``) to the remaining "active" pools in the deployment.
+MinIO supports decommissioning and removing a :ref:`server pools <minio-intro-server-pool>` from a deployment. 
```

Comment:
```suggestion
MinIO supports decommissioning and removing  :ref:`server pools <minio-intro-server-pool>` from a deployment with two or more pools. 
```

Feels like its stating the obvious, but sometimes the obvious isn't

Reply:
Indeed. 👍